content/applications/inventory_and_mrp/repairs/repair_orders.rst

```diff
@@ -0,0 +1,258 @@
+
+Finally, in the :guilabel:`Used` column, tick the checkbox once the part has been used in the repair
+process.
+
+To add additional columns to the line, click the :guilabel:`(slider)` icon, at the far-right of the
+header row. Select the desired options to add to the line.
+
+.. image:: repair_orders/repair-orders-additional-options.png
+   :align: center
+   :alt: Optional additional options to add to new part line.
+
+Repair Notes and Miscellaneous tabs
+-----------------------------------
+
+Click the :guilabel:`Repair Notes` tab to add internal notes about this specific |RO|, and anything
+the user performing the repair might need to know.
+
+Click the blank text field to begin writing notes.
+
+Click the :guilabel:`Miscellaneous` tab to see the :guilabel:`Operation Type` for this repair. By
+default, this is set to :guilabel:`YourCompany: Repairs`, indicating this is a repair type
+operation.
+
+Once all desired configurations have been made on the |RO| form, click :guilabel:`Confirm Repair`.
+This moves the |RO| to the :guilabel:`Confirmed` stage, and reserves the necessary components needed
+for the repair.
+
+A new :guilabel:`Forecasted` column appears on the product lines under the :guilabel:`Parts` tab,
+displaying the availability of all components needed for the repair.
+
+Once ready, click :guilabel:`Start Repair`. This moves the |RO| to the :guilabel:`Under Repair`
+stage (in the upper-right corner). If the |RO| should be canceled, click :guilabel:`Cancel Repair`.
+
+Once all products have been successfully repaired, the |RO| is completed. To register this in the
+database, click :guilabel:`End Repair`.
+
+.. note::
+   If all parts added to the |RO| were not used, clicking :guilabel:`End Repair` causes an
+   :guilabel:`Uncomplete Move(s)` pop-up window to appear.
+
+   .. image:: repair_orders/repair-orders-uncomplete-moves.png
+      :align: center
+      :alt: Uncomplete Moves pop-up window for unused parts.
+
+   The pop-up window informs the user that there is a difference between the initial demand and the
+   actual quantity used for the order.
+
+   If the :guilabel:`Used` quantity should be changed, click :guilabel:`Discard` or close the pop-up
+   window. If the order should be confirmed, click :guilabel:`Validate`.
+
+This moves the |RO| to the :guilabel:`Repaired` stage. A :guilabel:`Product Moves` smart button also
+appears above the form.
+
+Click the :guilabel:`Product Moves` smart button to view the product's moves history during and
+after the repair process.
+
+.. image:: repair_orders/repair-orders-product-moves.png
+   :align: center
+   :alt: Moves history of product included in the repair order.
+
+.. _repairs/repair_orders/return-customer:
+
+Return product to customer
+--------------------------
+
+Once the product has been successfully repaired, it can be returned to the customer.
+
+If the product is not under warranty, or should the customer bear the repair costs, click
```

Comment:
Since the instructions for products not under warranty is completely different then the steps that proceed after this section, consider putting an h4 heading here as well as line 218 to differentiate the two instructions as conditional flows.

Reply:
Nice catch - added H4 headings here to separate the instances of customer returns. Suggestion implemented ✅